acceptance/.kitchen.yml

```diff
@@ -0,0 +1,35 @@
+---
+driver:
+  name: vagrant
+  customize:
+    cpus: 4
+    memory: 2048
+
+provisioner:
+  name: chef_zero
```

Comment:
Don't you need to use chef_github here and kitchen-appbundle-updater gem in order to run the PR against the new chef code?  If we're assuming that the build has to go through all the way and generate an omnibus artifact, then I'd still expect to see configuration here that would pull that build out of artifactory (hooks in test-kitchen that I do not think exist yet).